pkg/unidling/util/scale.go

```diff
@@ -22,6 +21,8 @@ import (
 	unidlingapi "github.com/openshift/origin/pkg/unidling/api"
 )
 
+const legacyGroupName = ""
```

Comment:
do we still need this?

Reply:
> do we still need this?

Yeah.  Existing data is there and we aren't migrating it.